packages/main/src/themes/Input.css

```diff
@@ -254,12 +254,12 @@
 	border-color: var(--_ui5_input_value_state_error_border_color);
 }
 
-:host([value-state="Error"][focused]:not([readonly])) {
+:host([value-state="Error"][focused]:not([opened]):not([readonly])) {
 	background-color: var(--_ui5_input_focused_value_state_error_background);
 	border-color: var(--_ui5_input_focused_value_state_error_border_color);
 }
 
-:host([value-state="Error"][focused]:not([readonly])) .ui5-input-focusable-element:after {
+:host([value-state="Error"][focused]:not([opened]):not([readonly])) .ui5-input-focusable-element:after {
```

Comment:
I think that we do not need that enhancement (valid for all value state selectors below)